tests/test_cmake_config.py

```diff
@@ -9,10 +11,26 @@
 
 DIR = Path(__file__).parent.absolute()
 
+# Due to https://github.com/scikit-build/cmake-python-distributions/pull/279
```

Comment:
Not sure why this just hit this. Or why we didn't have to filter out `-GNinja` before.